apps/mail/components/mail/mail-list.tsx

```
@@ -968,7 +968,7 @@ export const MailList = memo(
                     <p className="text-lg">It's empty here</p>
                     <p className="text-md text-muted-foreground dark:text-white/50">
                       Search for another email or{' '}
-                      <button className="underline" onClick={clearFilters}>
+                      <button className="underline cursor-pointer" onClick={clearFilters}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)